{{cookiecutter.distribution_name}}/setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="{{cookiecutter.distribution_name}}",
+    version="{{cookiecutter.version}}",
```

Comment:
Good idea, we'll add the hack we're using in scikit-bio since we can't import directly.